addons/website/models/website.py

```diff
@@ -267,7 +267,7 @@ def configurator_init(self):
         company_id = self.get_current_website().company_id
         r['features'] = self.env['website.configurator.feature'].search_read([], ['name', 'description', 'type', 'icon', 'website_types_preselection'])
         r['logo'] = False
-        if company_id.logo and company_id.logo != company_id._get_logo():
+        if company_id.logo:
```

Comment:
What is the implication of this change? Is company logo actually using initials? I don't think this should be the case?

Reply:
Since we removed the _get_logo function, it doesn't make sense to call it. _get_logo was being used to set a default value to the logo, but now we have initials avatar on the logo as well

Reply:
But do we really want to use initials for the logo of a company?